### ISSUE TITLE
Update check a little broken

### ISSUE THREAD

**Issue by miiichael** [TheCBProject/EnderStorage#42]
Hello, I've just created a 1.11.2 pack (containing Ender Storage 1.11.2-2.3.1.116), and I'm ~~outraged~~ ~~dismayed~~ very slightly confused that Minecraft/Forge claims that there's an update available before the paint has even dried on my modpack. I assume it's because the mod and Forge have different ideas of what the mod version actually is, or is getting confused by looking to chickenbones.net for update information.

![image](https://user-images.githubusercontent.com/4179353/27915610-752fe9be-6299-11e7-9b78-dbbd73867f4f.png)

 Thanks!

**Comment by jcarrete5** [comment]
I am running into the same problem where the EnderStorage mod says it requires an update even though I have the latest version downloaded. I believe the issue is caused by a malformed mcmod.info file. I think that forge is failing to parse the mcmod.info because of the newlines in description field.
```
[
{
  "modid": "EnderStorage",
  "name": "EnderStorage",
  "description": "Stores your stuff in the END!    <-- here
                                                   <-- here
Credits: Ecu - original idea, design,              <-- here
        chest and pouch texture                    <-- here
    Rosethorns - tank model                        <-- here
    Soaryn - tank texture                          <-- here
                                                   <-- here
Supporters: Blkdragon112",
  "version": "${version}",
  "mcversion": "${mc_version}",
  "url": "http://www.minecraftforum.net/topic/909223",
  "authorList": [ "ChickenBones, covers1624" ],
  "dependencies": [ "CodeChickenCore" ]
}
]
```

**Edit:** This could also explain why the game is able to successfully load. I do not have the CodeChickenCore dependency so the game shouldn't even load up. Since the mcmod.info fails to be parsed, the game doesn't know about any dependencies the mod may need and the game loads and displays the warning shown in the above screenshot.

**Comment by miiichael** [comment]
I've also just noticed this warning from Forge buried in the logs:
```[21:57:35] [Server thread/WARN] [enderstorage/]: Mod enderstorage is missing the required element 'version' and no fallback can be found. Substituting '1.0'.```

**Comment by Hitel98** [comment]
I can confirm this, because i have the latest 1.12 version and it says, that i have 1.0 and the little green icon shows up saying that it needs update while it does'nt.

**Comment by covers1624** [comment]
This should be fixed now.

**Comment by miiichael** [comment]
I don't suppose there will be a fixed 1.11 release?

**Comment by covers1624** [comment]
Not at this stage, if I have enough things to back port, I'll consider it.
